src/triage/component/audition/distance_from_best.py

```diff
@@ -354,6 +357,11 @@ def plot_best_dist(metric, parameter, df_best_dist, **plt_format_args):
     cat_col = 'model_type'
     plt_title = 'Fraction of models X pp worse than best {} {}'.format(metric, parameter)
 
+    if directory:
+        path_to_save = os.path.join(directory + f'/distance_from_best_{metric}{parameter}.png')
```

Comment:
not a great use of `os.path.join` :stuck_out_tongue: 

you didn't switch the `+` for `, `